.gitignore

```diff
@@ -1,4 +1,8 @@
 # Cache files
 __pycache__/
 *.py[cod]
-*$py.class
\ No newline at end of file
+*$py.class
+.venv/
+.vscode/
```

Comment:
This should not go into the public `.gitignore`, but instead go into your local `.git/info/exclude` or equivalent